README.md

```diff
@@ -10,7 +10,7 @@ The CORE-V MCU DevKit is an evaluation platform for the OpenHW RISC-V CV32E40P0
 ## About the devkit
 
 This devkit has the following features:
-1. CV32E40P0 MCU + Embedded FPGA
+1. CV32E40P (v1.0.0) RISC-V core + Embedded FPGA
 2. 4MB Flash memory to hold programs and EFPGA bit images
 3. Onboard Ashling Opella LD JTAG debugger + Serial Consol interface
```

Comment:
```suggestion
3. Onboard Ashling Opella LD JTAG debugger 
4. JTAG connector for external debug access
5. USB-C for terminal and onboard debug access
```

Reply:
AFAIK, items 3 and 4 in your above list are the same connector.

Reply:
3 is the debugger, 4 is the connector